websockets/websockets.go

```diff
@@ -168,10 +169,17 @@ func defineWebsocket(rt *goja.Runtime, w *webSocket) {
 	// protocol
 	must(rt, w.obj.DefineAccessorProperty(
 		"binaryType", rt.ToValue(func() goja.Value {
-			return rt.ToValue("ArrayBuffer")
-		}), rt.ToValue(func() goja.Value {
-			common.Throw(rt, errors.New("binaryType is not settable in k6 as it doesn't support Blob"))
-			return nil // it never gets to here
+			return rt.ToValue(w.binaryType)
+		}), rt.ToValue(func(s string) error {
+			switch s {
+			case "blob":
+				return fmt.Errorf("blob is currently not supported, only arraybuffer is")
+			case "arraybuffer":
```

Comment:
I'd suggest moving to the `const` to quickly locate the places where either `arraybuffer` or `blob` have specific handling.

Reply:
done as part of [35f38ef](https://github.com/grafana/xk6-websockets/pull/64/commits/35f38efaa50920e270d888e37bfd5515a2f35c24)